pcsx2/gui/AppInit.cpp

```diff
@@ -243,6 +243,7 @@ void Pcsx2App::OnInitCmdLine( wxCmdLineParser& parser )
 	parser.AddSwitch( wxEmptyString,L"nohacks",		_("disables all speedhacks") );
 	parser.AddOption( wxEmptyString,L"gamefixes",	_("use the specified comma or pipe-delimited list of gamefixes.") + fixlist, wxCMD_LINE_VAL_STRING );
 	parser.AddSwitch( wxEmptyString,L"fullboot",	_("disables fast booting") );
+	parser.AddOption( wxEmptyString,L"gameargs",    _("passes the specified space-delimited string of launch arguments to the game"), wxCMD_LINE_VAL_STRING);
```

Comment:
Tabs instead of spaces after `gameargs",` the other lines use tabs.

Reply:
Ah, indeed.  Fixed it.